english/stargate/circle/JungleWorld.md

```diff
@@ -0,0 +1 @@
+Entering in the symbol for Circle, the gate spins and locks in the last symbol. You enter and arrive on a Jungle world and are immediately surrounded by raptor like creatures eyeing you as their next meal.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.